sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/SchemaCoder.java

```diff
@@ -59,7 +60,7 @@
 @Experimental(Kind.SCHEMAS)
 public class SchemaCoder<T> extends CustomCoder<T> {
   // This contains a map of primitive types to their coders.
-  public static final ImmutableMap<TypeName, Coder> CODER_MAP =
+  public static final Map<TypeName, Coder> CODER_MAP =
```

Comment:
This change is required because we should not be exposing Guava (even our own vendored version) in the public API and it was surfaced by `GcpCoreApiSurfaceTest` break